### ISSUE TITLE
tested with gutenberg

### ISSUE THREAD

**Issue by merijnvanessen** [JamesParkNINJA/acf-repeater-tabs#22]
hi,
is this plugin tested with gutenberg yet?
It doesnt seem to work, or I'm doing something wrong

registered a block with acf_register_block but no tabs appears

![schermafbeelding 2019-01-15 om 22 48 10](https://user-images.githubusercontent.com/23075300/51212102-cdd8f200-1917-11e9-8b74-8db368932153.png)

**Comment by JamesParkNINJA** [comment]
It hasn't been tested with Gutenberg yet sadly, I'm still working on the Classic Editor at the moment until I have the spare time to look into it further.

**Comment by Blackout8099** [comment]
I don't think this is a result of using Gutenberg. I had the same display although this was fixed by editing the below:
jpn_acf_tabs.css line 54
`acf_version = parseFloat(jpn_acf_tabs_args.acf_version);`
to 
`acf_version = parseFloat(acf.get('acf_version'));`
